tutorials/throttle/src/main/java/com/datatorrent/examples/throttle/SlowDevNullOperator.java

```diff
@@ -0,0 +1,35 @@
+package com.datatorrent.examples.throttle;
+
+import com.google.common.base.Throwables;
+
+import com.datatorrent.api.DefaultInputPort;
+import com.datatorrent.common.util.BaseOperator;
+
+/**
+ * Created by pramod on 9/27/16.
```

Comment:
Ok for this repo, seen it other example applications like cassandra. Can be removed if example is moved to apex.